web/components/requests/FcSummaryStudyRequest.vue

```diff
@@ -1,38 +1,19 @@
 <template>
```

Comment:
in this file, and also in `<FcSummaryStudyRequestBulk>`: we should add the "Standard times to request counts are 2-3 months." message for non-urgent requests, as this allows these summaries to have more predictable spacing.